cmake/LICENSE

```diff
@@ -0,0 +1,39 @@
+Copyright (c) 2016, Los Alamos National Security, LLC
```

Comment:
Yes, I will update the year soon. If we wait a few months, we can update 2 years in a single step. ;-)